src/Database/LSMTree/Internal/RunBuilder.hs

```diff
@@ -285,6 +285,8 @@ copyBlob ::
 copyBlob builder@RunBuilder {..} blobref = do
     blob <- BlobRef.readRawBlobRef runBuilderHasFS blobref
     writeBlob builder blob
+    --TODO: consier adding write blob functions to BlobFile
+    -- variants: at offset, at file pointer with CRC update.
```

Comment:
I think we should probably do that. It also adds boilerplate, but it means the functionality is concentrated in the `BlobFile` module instead of throughout other code.

```suggestion
    -- TODO: consier adding write blob functions to BlobFile
    -- variants: at offset, at file pointer with CRC update.
```

Reply:
And I could consider spelling consider correctly.

Reply:
Ok, I've added a `writeBlob` but I can only easily use it in WriteBufferBlobs, not in RunBuilder, because it does everything uniformly via `ForRunFiles (ChecksumHandle (PrimState m) h)`